backend/doi/link_header_internal_test.go

```diff
@@ -0,0 +1,19 @@
+package doi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseLinkHeader(t *testing.T) {
+	header := "<https://zenodo.org/api/records/15063252> ; rel=\"linkset\" ; type=\"application/linkset+json\""
+	links := parseLinkHeader(header)
```

Comment:
suggestion: it is probably a good idea to add one more test case that has more than one link in the link header.